### PR TITLE
Running documentation in Docker

### PR DESCRIPTION
**Pull request by vochicong**
Use Docker to easily run the documentation locally.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ permalink:      pretty
 # Server
 source:         "site"
 destination:    ./_gh_pages
-host:           "localhost"
+host:           "0.0.0.0"
```
Comment (XhmikosR) [inline _config.yml]: This was intentional.

**Comment by XhmikosR** [comment]
I see no value for this, especially since everyone can do it themselves.

**Comment by vochicong** [comment]
Everyone can do it, but it takes each about 10 minutes or more.

File: _config.yml (hunk 1 above)
Comment (vochicong) [reply _config.yml]: `localhost` won't work for someone using Docker.

**Comment by mdo** [comment]
When none of the development of Bootstrap happens with Docker currently, why would we add this?

**Pull request closed without merging**